ldms/src/sampler/zfs_topvdevs/zfs_topvdevs.man

```diff
@@ -0,0 +1,62 @@
+.\" Manpage for Plugin_procnetdev2
```

Comment:
Replace "procnetdev2" with proper name everywhere in this man page. Maybe the whole man page still needs to be written for you plugin?